xml/System.Net.Security/SslStream.xml

```diff
@@ -3453,9 +3453,17 @@ Authentication has not occurred.</exception>
         <ReturnType>System.Net.Security.TlsCipherSuite</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the cipher suite resolved to use by this <see cref="T:System.Net.Security.SslStream"/>.</summary>
+        <value>The cipher suite resolved to use by this <see cref="T:System.Net.Security.SslStream"/>.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks
+This property gets the cipher suite that is going to be used in the communication between a client and server by the underlying SSL/TLS implementation. This property is only available after a successful call to <xref:System.Net.Security.SslStream.AuthenticateAsServer%2A> or <xref:System.Net.Security.SslStream.AuthenticateAsClient%2A>.
```

Comment:
what's the `%2A` for in the xref?

Reply:
URL-encoded star (*), I think is used to refer to the method's name without referring to a specific signature.

Reply:
> >what's the %2A for in the xref?
>
>  I think is used to refer to the method's name without referring to a specific signature.

Yeah, targets the method group (full overload set) rather than a specific method.

Reply:
It's when you want to link to a method in general and not a specific overload. More info at https://github.com/dotnet/docs/blob/master/styleguide/template.md#links-to-apis